src/@types.ts

```diff
@@ -1,20 +1,63 @@
-export interface RawSchema {
+import { Resolver, ForSchemaOptions } from 'avsc'
+
+export enum SchemaType {
+  AVRO = 'AVRO',
+  JSON = 'JSON',
+  PROTOBUF = 'PROTOBUF',
+  UNKNOWN = 'UNKNOWN',
+}
+
+export interface SchemaHelper {
+  validate(schema: Schema): void
+  getSubject(confluentSchema: ConfluentSchema, schema: Schema, separator: string): ConfluentSubject
+}
+
+export type AvroOptions = Partial<ForSchemaOptions>
+export type JsonOptions = any // FIXME:
+export type ProtoOptions = any // FIXME:
```

Comment:
This one is a little trickier. Currently we are expecting an object like `{ messageName: string }`, which doesn't map to any options in Protobufjs itself.

Given that Protobufjs doesn't have any constructor options or anything that we could use here, I suggest that we make this just `{ messageName: string }`, and then in the future if people have a need to pass `IParseOptions` or  something we can extend the type like `{ messageName: string, parse: IParseOptions }` etc.